src/SFML/Window/Window.cpp

```diff
@@ -58,7 +58,10 @@ Window::Window(WindowHandle handle, const ContextSettings& settings)
 
 
 ////////////////////////////////////////////////////////////
-sf::Window::Window(Window&&) noexcept = default;
+sf::Window::Window(Window&& other) noexcept : WindowBase(std::move(other)), GlResource(std::move(other))
+{
+    other.close();
```

Comment:
I think we can retain default move operations on `sf::Window`, right? When moved-from the `unique_ptr` inside is set to null which is already all of what `sf::Window::close` was doing before. The move constructor for the base class ought to be automatically invoked. I'm not totally sure about this all though.

Reply:
I added only added close for future maybe something else added in close function. but you are right it can also be default.